perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -329,7 +330,7 @@ def RunForMinutes(proc, mins_to_run, mins_per_call):
 
 def GetConfig(user_config):
   config = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
-  if FLAGS.against_device or FLAGS.prefill_device:
+  if FLAGS.fio_target_mode != 'against_file_without_fill':
```

Comment:
Sorry, I fixed it in another part of the file and missed this one. This is done now.